src/layout/Address/config.ts

```diff
@@ -15,7 +15,35 @@ export const Config = new CG.component({
     new CG.trb({
       name: 'title',
       title: 'Title from Summary',
```

Comment:
This is no longer just for summary, right?

Reply:
Correct, fixed :)